src/components/analyses/PendingTerminationDlg.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @author aramsey
+ *
+ * A dialog to inform users that analysis output folders are not created
+ * until after the job has terminated.
+ */
+
+import React from "react";
+
+import { build } from "@cyverse-de/ui-lib";
+import { Button, Typography } from "@material-ui/core";
+
+import DEDialog from "components/utils/DEDialog";
+import { useTranslation } from "i18n";
+import ids from "./ids";
+
+function PendingTerminationDlg(props) {
```

Comment:
Not sure if "Pending Termination" makes sense to anybody else - open to suggestions on the name